projects/components/src/header/application/application-header.component.ts

```diff
@@ -12,12 +12,16 @@ import { GLOBAL_HEADER_HEIGHT, NavigationService } from '@hypertrace/common';
           <ng-content select="[logo]"></ng-content>
         </div>
         <div class="divider"></div>
+        <div class="left-side-content">
+          <ng-content select="[left]"></ng-content>
+        </div>
         <div class="time-range">
           <ht-time-range></ht-time-range>
         </div>
       </div>
-      <div class="right-side">
+      <div class="right-side-content">
         <ng-content></ng-content>
+        <ng-content select="[right]"></ng-content>
```

Comment:
How is the `select="[right]"` property used here

Reply:
This was a relabel of the existing one. Previously, header controls added via content always went to the right. Since I added a way to add them to the left too, I disambiguated via the `left` attribute, I relabeled the previously default selector to use `right` for clarity.

Reply:
Still didn't get how two ng-content with one containing a select on `[right]` is working. What does this selector '[]'  means? I guess one of its child should have this selector defined somewhere.

Reply:
Fine to merge. I will play around with to understand what's happening here.

Reply:
Discussed offline (it's an attribute selector for header content, not yet used in this PR)